src/js/main.js

```diff
@@ -66,7 +66,7 @@ Site.SplashVideo = {
       _this.canAutoplay = true;
 
       // Set threshold
-      _this.threshold = _this.$video1.offset().top - ($(window).height() / 3) * 2;
+      _this.threshold = (_this.$video1.offset().top / 6);
```

Comment:
Why go back to what feels an arbritary figure? This isnt much different from just triggering on the first scroll movement as it basically means 'after scrolling a little bit'. Also this treshold doesnt seem to be possible to retrigger when scrolling back up the page. Is this just a debugging mistake as nothing else is changed?